_posts/2018-06-22-dear-new-engineers.md

```diff
@@ -0,0 +1,56 @@
+As a perfect way to end my co-op, I had to hotfix an issue that I released to production on my last day.
+I learned that everyone makes mistakes and the fear of making a mistake should never discourage you as a developer.
+The important part of making mistakes is owning up to them and learning from them to make sure they don't happen again.
+Here at Forward Financing we have a [blameless culture](http://tech.forwardfinancing.com/engineering/culture/2017/09/30/our-engineering-culture-methodologies-and-philosophies.html){:target='_blank'} where we all understand that bugs will happen and we won't single out the individual that committed the bug.
+By taking the blame off of the individual, it encourages growth as a group because we all learn from each other's mistakes one box of donuts at a time!
+
+# Step outside your comfort zone
+Before I started, I really had no idea whether I wanted to do front end or back end development.
+Luckily, every engineer at Forward Financing is a full stack engineer.
+Though each engineer has freedom in choosing what kind of tasks he or she wants to do, everyone here does their fair share of front end and back end work.
+As someone who was unsure about which direction to go in, being a full stack engineer is especially helpful for figuring out what to do.
+Prior to working here, I was always more comfortable doing back end work because it would not be immediately visible to users.
+However, when I started as a full stack engineer, I was eager to try out all different types of tasks, especially in languages or frameworks I was unfamiliar with.
+I found that diving into task I was unfamiliar with was the best way for me to learn and grow as an engineer.
+
+# Stay learning
+We are part of an industry that is constantly evolving with new technologies and frameworks popping up and changing the way we work.
+It is important to keep up with all the changes that seem to happen every week in our industry.
+I quickly realized that no matter how long someone has been in the industry, there is a constant stream of new information and new ideas to learn.
+Being new to the industry, I found myself spending just as much time on the job learning as I did actually coding.
+Within my first month, I learned to use React to work on our front-end apps and Elixir to create new API endpoints by simply immersing myself into our codebase and taking advantage of the extensive resources I had at hand, whether it be other people or communities online.
+Whenever I find myself with a natural break in the workday, like waiting for code reviews, I'm constantly reading articles and documentation to learn about technologies that we are using here at Forward Financing or new ones that I am curious about working with.
+It feels great to learn be constantly learning because it keeps your day to day work from ever getting stale and it ensures that you are continuously growing.
```

Comment:
@kyle-ch Remove `learn` from above sentence as it follows with `constantly learning`?